org-formation/organization.yaml

```diff
@@ -50,6 +50,7 @@ Organization:
         - !Ref SecurityCentralAccount
         - !Ref ImageCentralAccount
         - !Ref ITSandboxAccount
+        - !Ref ContainerCentralAccount
```

Comment:
Is the plan that scipool users will register containers? I wonder if the `ContainerCentral` account should go in a ScienceOU rather than an ITOU... since non-IT users will be registering things in it

Reply:
let's figure out OU and account placement in PR #123